examples/with-styledcomponent/app/entry.server.tsx

```diff
@@ -0,0 +1,39 @@
+import ReactDOMServer,{renderToString} from "react-dom/server";
```

Comment:
The README.md should reference this file because it's relevant.

Might be useful to put some code comments in this file as well to describe what's going on and why different bits are needed.

Reply:
sure will be doing that

Reply:
I have done that please check